app/src/main/java/com/hous/hous_aos/ui/rules/RulesViewModel.kt

```diff
@@ -33,14 +33,66 @@ class RulesViewModel : ViewModel() {
         MutableLiveData<List<GeneralRulesData>>()
     val generalRulesTableList get() = _generalRulesTableList
 
-    init {
-        fetchToTodayToDoList()
-        fetchToMyTodayToDoList()
-        fetchToCategoryOfRuleList()
+    private var _tmpManagerList = MutableLiveData<List<HomieData>>()
+    val tmpManagerList get() = _tmpManagerList
+
+    /**
+     * 임시 담당자 checked 바꾸기
+     * */
+    fun setSelectedTmpManager(position: Int) {
+        val flag = requireNotNull(_tmpManagerList.value)[position].isChecked
+        requireNotNull(_tmpManagerList.value)[position].isChecked = !flag
+    }
```

Comment:
```suggestion
    fun setSelectedTmpManager(position: Int) {
        requireNotNull(_tmpManagerList.value)[position].isChecked = !requireNotNull(_tmpManagerList.value)[position].isChecked
    }
```
변수를 안써도 할 수 있지 않을까 생각이 드네요

Reply:
오호라 감사함닏다